### DISCUSSION TITLE
Server Error

### DISCUSSION THREAD

**Discussion by mihaimazareanu** [neuefische/questions#358] [Web] [answered]
Hi,

I've created a DB on MongoDB and I connected with it through Compass, but when I am trying to connect my app with it, I get the following error:

    Server Error
    Error: Please set environment variable MONGO_DB_URI.

This is the message that is set in `connect-db.js` to show when the MONGO-DB_URI is not provided, but I have added this in `.env.local` 

So what am I doing wrong?

This is how my `.env.local` looks like:

    # MONGO_DB_URI=mongodb://localhost:27017/kidsFi

    MONGO_DB_URI=mongodb+srv://mihaimazareanu:<password>@cluster0.guv1k9s.mongodb.net/test

I have inserted the correct password instead of `<password>` I took it out for safety purposes.

The code for MONGO_DB_URI I got from MongoDB.

Please help... 🙏

**Comment by pixelass** [comment]
Is this happening on your local environment or on a deployment?

**Reply by mihaimazareanu** [reply]
Local environment

**Comment by actyralf** [comment, marked as answer]
I checked with Mihai - `.env.local` was not located in the root directory.
